src/components/staking/Summary.svelte

```diff
@@ -202,6 +212,22 @@
           isCompounding = false;
         });
     }
+
+    const checkNotarization = async () => {
+        const merkleTreeDistributorContract = getMerkleTreeDistributorContract($eth);
+        const nextCreatedIndex = await merkleTreeDistributorContract.nextCreatedIndex();
+        const currentWindowIndex = nextCreatedIndex.sub(BigNumberEthers.from(1))
+        const currentWindow = await merkleTreeDistributorContract.merkleWindows(currentWindowIndex);
+        isNotarizing = currentWindow.merkleRoot !== ZERO_ADDRESS;
+        snapshotAddress = currentWindow.ipfsHash;
+    }
+
+    onMount(async () => {
+        const res = await fetch("https://raw.githubusercontent.com/pie-dao/shared-metadata/main/slice-rate.json");
+        const data = await res.json();
+        nextRate = data.nextRate;
+        nextCompoundWindow = new Date(data.nextCompoundWindow).toLocaleDateString();
```

Comment:
Might want to error handle as this could blow up - not sure what the fallback here is.

Reply:
Good spotted